File: app/src/main/java/com/brookmanholmes/drilltracker/presentation/model/DrillPackModel.java
package com.brookmanholmes.drilltracker.presentation.model;

import androidx.annotation.NonNull;

/**
 * Created by Brookman Holmes on 8/9/2017.
 */

public class DrillPackModel extends Model {
    public String name;
    public String price;
    public String description;
    public String sku;
    public String url;
    public boolean purchased = false;
    public String token;


    @NonNull
    @Override
    public String toString() {
        return "DrillPackModel{" +
                "name='" + name + '\'' +
                ", price='" + price + '\'' +
                ", description='" + description + '\'' +
                ", sku='" + sku + '\'' +
                ", url='" + url + '\'' +
                ", purchased=" + purchased +
                ", token='" + token + '\'' +
                '}';
    }
}

File: app/src/main/java/com/brookmanholmes/drilltracker/presentation/model/DrillPackModel.kt
package com.brookmanholmes.drilltracker.presentation.model

/**
 * Created by Brookman Holmes on 8/9/2017.
 */
data class DrillPackModel(
    val id: String,
    val name: String,
    var price: String,
    val description: String,
    val sku: String,
    val url: String?,
    var purchased: Boolean,
    var token: String
) : Model {
    override fun getModelId(): String {
        return id
    }
}